docs/modules/develop/pages/guide_development_with_localhost_ssl.adoc

```diff
@@ -1,47 +1,44 @@
 = Testing SSL in localhost
 
-In some cases is useful to access the `localhost` environment under SSL (ie: `https://localhost:3000/`).
+In some cases is useful to access the `localhost` environment under SSL (ie: `https://localhost:3443/`).
 
 A typical use case is when you want to test login integration with 3rd party systems using OAuth (which in the latest versions only the HTTPS protocol is allowed).
 
-Using Puma, the default server for Rails, you can easily do the trick by creating a self-signed certificate and using it when starting your development environment.
+Using Puma, the default server for Rails, you can easily do the trick by creating a self-signed certificate and using it when starting your development environment. A dummy certificate is shipped within the `decidim-dev` module which is usually included in the `:development` group of your `Gemfile`.
 
-== Create a self-signed certificate for localhost
+== Starting development in SSL mode
 
-First you need a SSL certificate for the `localhost` domain (you only need to execute this command once):
+If you created the Decidim development app using the default development app generator, you already should have the necessary configurations for starting the development server under SSL. If you created the development app using older versions of the Decidim generators, you need to add the following code at the end of your `config/puma.rb`:
```

Comment:
I would skip the whole "If you created the development app using older versions" thing. IMHO development_ppp should be regenerated periodically, as the generator is changing. Also documentation need to be maintained too and it's easy that we skip updating this if we make changes on the generator. 

This is what I had in mind:  

> 
> If you created the Decidim development app using the default development app generator, you already should have the necessary configurations for starting the development server under SSL. You can start the development server under SSL at port 3443 (default) using the following command:
> 
> ```console
> DEV_SSL=true bin/rails s
> ```

Reply:
I've also done this change in the doc.

Also the instance apps are created using this command, so I thought that it might help with older instances that got upgraded on the way since we are not touching this file during the upgrade process.

Anyways, it's now as requested.